### DISCUSSION TITLE
Is it safe to depend on cards when its labelled as Experimental?

### DISCUSSION THREAD

**Discussion by distortedMango** [rstudio/bslib#968] [Q&A] [answered]
Hi,

We are about to build a Shiny App, and are planning to use `bslib` - but based on the documentation some of the core functions like the `card()`-function is labelled as `experimental`, which makes us think twice. 

Is there any expected breaking changes? Or how is this being managed?

Best,

**Comment by cpsievert** [comment, marked as answer]
Yes, they are safe to use, and we'll likely be removing those labels in the near future. We just want to be transparent that we reserve the right to make changes to the API, but we also don't have any immediate plans to do. Also, when we do make changes, we always provide instructions for moving to the new API in the [`NEWS.md` file](https://github.com/rstudio/bslib/blob/main/NEWS.md).
